zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/incident/ElasticsearchIncidentUpdateRepository.java

```diff
@@ -84,24 +119,16 @@ private SearchRequest createPendingIncidentsBatchRequest(final int size, final Q
 
   private Query createPendingIncidentsBatchQuery(final long fromPosition) {
     final var positionQ =
-        QueryBuilders.range()
-            .field(PostImporterQueueTemplate.POSITION)
-            .gt(JsonData.of(fromPosition))
-            .build()
-            ._toQuery();
+        QueryBuilders.range(
+            r -> r.field(PostImporterQueueTemplate.POSITION).gt(JsonData.of(fromPosition)));
     final var typeQ =
-        QueryBuilders.term()
-            .field(PostImporterQueueTemplate.ACTION_TYPE)
-            .value(f -> f.anyValue(JsonData.of(PostImporterActionType.INCIDENT)))
-            .build()
-            ._toQuery();
+        QueryBuilders.term(
+            t ->
+                t.field(PostImporterQueueTemplate.ACTION_TYPE)
+                    .value(PostImporterActionType.INCIDENT.name()));
```

Comment:
:question: is there any difference in using `value` vs `anyValue`?

Reply:
AFAIK it's just a short cut, but I didn't dig too much into it. I mostly lifted it from the old code anyway, as I wasn't sure how the client will serialize an enum.